visualization/app/codeCharta/ui/legendPanel/legendScreenshotButton/legendScreenshotButton.component.ts

```diff
@@ -47,7 +47,7 @@ export class LegendScreenshotButtonComponent {
 		return document.getElementById("legend-panel")
 	}
 
-	private async captureScreenshot(element: HTMLElement): Promise<HTMLCanvasElement> {
+	async captureScreenshot(element: HTMLElement): Promise<HTMLCanvasElement> {
```

Comment:
Why was the private here removed? If it is just for testing purposes im sure there is a better way to solve this